app/models/institution.rb

```diff
@@ -7,18 +10,19 @@ class Institution < ApplicationRecord
   mount_uploader :image, ImageUploader
   mount_uploader :hero_image, ImageUploader
 
-
   has_many :collections
   has_many :transcription_conventions
 
   validates :name, presence: true
   validates :name, uniqueness: true
-  validates :slug, format:  { with: /\A^[a-zA-Z0-9-]*$\z/ }
+  validates :slug, format: { with: /\A^[a-zA-Z0-9-]*$\z/ }
   validates :max_line_edits, numericality: true
 
+  attribute :max_line_edits, :integer, default: 3
+
   validate :image_size_restriction
 
-  scope :order_asc, -> { order("LOWER(institutions.name)") }
+  scope :order_asc, -> { order('LOWER(institutions.name)') }
```

Comment:
Can you please use https://github.com/reinteractive/reinteractive-style?
Reinteractive default is double quotes :)